core-web/libs/edit-content/src/lib/components/dot-edit-content-sidebar/components/dot-edit-content-sidebar-locales/dot-edit-content-sidebar-locales.component.scss

```diff
@@ -0,0 +1,42 @@
+@use "variables" as *;
+
+:host {
+    display: flex;
+    gap: $spacing-1;
+    padding-top: 0.15rem;
+
+    ::ng-deep {
+        p-chip {
+            position: relative;
+
+            .p-chip {
```

Comment:
@hmoreras I think we should avoid overriding or creating new styles, such as `translated` or `current`. Instead, I suggest using our Design System. For example, by leveraging existing severity properties and styleClass, it is possible to achieve the same styling.

```html
<p-chip severity="p-chip-primary" styleClass="p-chip-filled">
    en-US
</p-chip>
<p-chip severity="p-chip-primary">
    en-es
</p-chip>
<p-chip styleClass="p-chip-white">
    it-it
</p-chip>
```

![Screenshot 2024-11-22 at 3 34 28 PM](https://github.com/user-attachments/assets/b967f689-1c4d-45f2-9953-7ae4ce22d876)

I think the missing case is the dashed border. In this situation, we should improve the chip styles to accommodate this new case.

Ref: https://www.figma.com/design/7CzbMcGSbTIJjerWtcapAm/Design-System-2023?node-id=808-11280
Storybook: http://localhost:4400/?path=/docs/primeng-misc-chip--docs

Reply:
Maybe something like: 
```
[ngClass]="{
            primary: locale.translated,
            secondary: contentlet.languageId === locale.id,
            no-idea: defaultLocale.id === locale.id
        }
```
that is it.